pkg/commands/parse/parse_test.go

```diff
@@ -0,0 +1,78 @@
+package parse
```

Comment:
It would be nice to also include a test to check if valid JSON is returned by the command

Reply:
I replaced some tests with JSON assertions.